outplacement_partner_ssn/views/ir_actions_server_view.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="load_ais-f_data" model="ir.actions.server">
+        <field name="name">Load social security numbers</field>
+        <field name="model_id" ref="base.model_ir_actions_server"/>
+        <field name="state">code</field>
+        <field name="code">
+partners = env['resp.partner'].search([('company_registry', '!=' False)])
```

Comment:
@SebastianGieser-Vertel 
Are you sure it should be `env['resp.partner']` instead of `env[res.partner]` ?

Reply:
this is why we have code reviews